casbin/enforcer_interface.h

```diff
@@ -60,10 +60,11 @@ class IEnforcer {
         virtual bool EnforceWithMatcher(string matcher, Scope scope) = 0;
 
         /* RBAC API */
-        virtual vector<string> GetRolesForUser(string name) = 0;
-        virtual vector<string> GetUsersForRole(string name) = 0;
+        virtual vector<string> GetRolesForUser(string name, vector<string> domain) = 0;
```

Comment:
Can we make `domain` into an optional arg?

Reply:
Yes, we can do that.

Reply:
I think Golang Casbin makes it optional. Please follow the similar pattern.